aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -129,6 +129,10 @@ module aptos_framework::account {
 
     native fun create_signer(addr: address): signer;
 
+    #[test_only]
+    /// Create signer for testing, independently of an Aptos-style `Account`.
+    public fun create_signer_for_test(addr: address): signer {create_signer(addr)}
```

Comment:
Add spaces after`{` and before `}`.

Reply:
@alinush 7612b0740ceb9ff1534c3ab4ce0d881f87b87270 adds space padding per your request.